### DISCUSSION TITLE
string_agg() function not working

### DISCUSSION THREAD

**Discussion by ildikoszeles** [dbeaver/dbeaver#34842] [Q&A] [answered]
### Description

Hi,
I cannot use the string_agg() function in an MS SQL Server database. Can you help me in this issue, please?
Error message: SQL Error [195] [S0010]: 'string_agg' is not a recognized built-in function name.
Thanks a lot in advance!
Regards,
Ildiko

### DBeaver Version

Community Edition 24.1.0

### Operating System

Win 10

### Database and driver

Driver: Microsoft JDBC Driver for SQL Server

**Comment by E1izabeth** [comment]
Could you tell me your driver and server version? You can get this information by clicking `Test connection...` button in the `Connection configuration` dialog.
![image](https://github.com/user-attachments/assets/9fc67fd6-bce1-4c57-b575-c7eb536a59b9)

**Comment by E1izabeth** [comment, marked as answer]
Here is the documentation on this matter https://learn.microsoft.com/en-us/sql/t-sql/functions/string-agg-transact-sql?view=sql-server-ver16
It says, that this function was added since SQL Server 2017

**Comment by ildikoszeles** [comment]
Hi Elizabeth,
![image](https://github.com/user-attachments/assets/c0d9ef36-d2ae-49c8-a338-8c5f1b51fe2e)
thanks for your helping!

**Comment by E1izabeth** [comment]
Yeah, there is no such function for your MS SQL Server version.

**Comment by ildikoszeles** [comment]
Yes, then I go to my IT colleagues...
Thanks and have a nice day!
